package.json

```diff
@@ -4,7 +4,7 @@
   "description": "openaps oref0 reference implementation of the reference design",
   "scripts": {
     "test": "make test",
-    "global-install": "npm install && sudo npm install -g && sudo npm link && sudo npm link oref0"
+    "global-install": "npm install && sudo npm link && sudo npm link oref0 && sudo npm install -g"
```

Comment:
What does changing this order do?

Reply:
For me it fixed #1145 by allowing global-install to work